dmsrc/world.dm

```diff
@@ -0,0 +1,3 @@
+/world/Del(...)
+	RUSTG_CALL(RUST_G, "shutdown_rustg")
```

Comment:
This'll end up just copying itself  in the rust_g.dm file right? It needs documentation on what it's doing and why it's doing it then.
Some codebases are particular about ordering. See: you don't want this before the `logs << "shutting down"` logging entry.